### PR TITLE
Delete and export

### LINKED ISSUE #22: Restore download dataset button
Downloading a dataset hasn't worked post-white paper refactor. Time to make that work again.

### LINKED ISSUE #98: Check that download and delete datasets work

### PR DESCRIPTION
**Pull request by ramfox**
- clicking 'export' downloads a zip of the dataset to your downloads folder
- clicking 'delete' should delete the dataset and redirect you to the console

closes #22 
closes #98

### CODE REVIEW

File: app/actions/dataset.js [hunk 1]
```diff
@@ -373,13 +373,13 @@ export function loadDatasetChanges (datasetId, page = 1, pageSize = 50) {
   }
 }
 
-export function downloadDataset (address = '') {
+export function downloadDataset (hash = '') {
```
Comment (b5) [inline app/actions/dataset.js]: is this action in use at all anymore?

**Review by b5 — APPROVED** [review]
🏄‍♀️

File: app/actions/dataset.js (hunk 1 above)
Comment (ramfox) [reply app/actions/dataset.js]: No, but I was too chicken to delete it.

Should I delete it?

Comment (b5) [reply app/actions/dataset.js]: yea yea feel free to delete!  in a later PR, obvi)

**Pull request merged**